### ISSUE TITLE
Not working ubuntu 24.04

### ISSUE THREAD

**Issue by mraicunha** [panaceya/winbox#47]
I'm testing Ubuntu 24.04. Winbox snap isn't work.

journalctl: 

Apr 18 10:21:56 mrai-noc systemd[2621]: Started snap.winbox.winbox-82307a99-1f85-41bf-8625-fdf97e4b276d.scope.
Apr 18 10:21:57 mrai-noc kernel: audit: type=1326 audit(1713446517.390:359): auid=1000 uid=1000 gid=1000 ses=3 subj=snap.winbox.winbox pid=32832 comm="wineserver" exe="/snap/winbox/135/wine-platform/wine-stable/bin/wineserver" sig=0 arch=c000003e syscall=203 compat=0 ip=0x7621626b3f09 code=0x50000
Apr 18 10:21:57 mrai-noc winbox_winbox.desktop[32834]: 000b:err:module:__wine_process_init failed to load L"C:\\windows\\system32\\wineboot.exe", error c0000018
Apr 18 10:21:57 mrai-noc winbox_winbox.desktop[32829]: 0009:err:environ:run_wineboot failed to start wineboot c0000018
Apr 18 10:21:57 mrai-noc kernel: audit: type=1400 audit(1713446517.682:360): apparmor="DENIED" operation="open" class="file" profile="snap.winbox.winbox" name="/run/" pid=32832 comm="wineserver" requested_mask="r" denied_mask="r" fsuid=1000 ouid=0
Apr 18 10:21:57 mrai-noc kernel: audit: type=1326 audit(1713446517.687:361): auid=1000 uid=1000 gid=1000 ses=3 subj=snap.winbox.winbox pid=32832 comm="wineserver" exe="/snap/winbox/135/wine-platform/wine-stable/bin/wineserver" sig=0 arch=c000003e syscall=203 compat=0 ip=0x7621626b3f09 code=0x50000
Apr 18 10:21:57 mrai-noc kernel: audit: type=1400 audit(1713446517.790:362): apparmor="DENIED" operation="connect" class="file" profile="snap.winbox.winbox" name="/run/cups/cups.sock" pid=32836 comm="winbox64.exe" requested_mask="wr" denied_mask="wr" fsuid=1000 ouid=0
Apr 18 10:21:57 mrai-noc winbox_winbox.desktop[32836]: 000d:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
Apr 18 10:21:57 mrai-noc winbox_winbox.desktop[32836]: 000d:err:winediag:nodrv_CreateWindow The explorer process failed to start.

**Comment by gil-obradors** [comment]
Hi!
The same thing happens to me
```
000b:err:module:__wine_process_init failed to load L"C:\\windows\\system32\\wineboot.exe", error c0000018
0009:err:environ:run_wineboot failed to start wineboot c0000018
➜  ~ 000d:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
000d:err:winediag:nodrv_CreateWindow The explorer process failed to start.
```

snaps installed:
```
winbox                     3.40                   135    latest/stable    thepanaceya  -
wine-platform-5-stable     5.0.3                  18     latest/stable    mmtrt        -
wine-platform-runtime      v1.0                   367    latest/stable    mmtrt        -

```

**Comment by mfth12** [comment]
me too
![image](https://github.com/panaceya/winbox/assets/53788525/7cb12d78-f443-4089-96cb-31df9153670a)

**Comment by panaceya** [comment]
Fixed in #53. 
You need before refresh by `snap refresh winbox --stable`.
